Auth/FirebaseAuthUI/FUIAuthBaseViewController.m

```diff
@@ -287,6 +288,15 @@ + (void)showAlertWithTitle:(nullable NSString *)title
     window.windowLevel = UIWindowLevelAlert + 1;
     [window makeKeyAndVisible];
     [viewController presentViewController:alertController animated:YES completion:nil];
+	  
+    if (@available(iOS 13.0, *)) {
+        /*
+            Earlier iOS versions established a strong reference to the window when makeKeyAndVisible was called.
+            Now we add one from the alert controller, to prevent objects from getting garbage collected right away.
+        */
+        static void *key;
+        objc_setAssociatedObject(alertController, key, window, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
```

Comment:
I think using `key` here is technically undefined behavior since `key` is uninitialized. Consider using a selector instead, or setting `key` to `nil` and using `&key`.